src/train.py

```diff
@@ -144,18 +146,21 @@ def main(argv):
             # Zero the parameter gradients for the discriminator
             optimizerDisc.zero_grad()
 
+            # Make predictions
+            predictionReal = disc(font)
+            predictionFake = disc(charsGenerated)
+            predictionCombined = torch.cat((predictionReal, predictionFake), 0)
+
             # Train the discriminator
-            prediction = disc(font)
-            lossDisc = criterionDisc(prediction, labels)
+            lossDisc = criterionDisc(predictionCombined, labels)
```

Comment:
You also need to do a concat for labels. Right now labels just contains the real.